nodes/gaussian_blur.py

```diff
@@ -13,8 +13,8 @@ def INPUT_TYPES(s):
         return {
             "required": {
                 "images": ("IMAGE",),
-                "kernel_size": ("INT", {"default": 61}),
-                "sigma": ("INT", {"default": 5}),
+                "kernel_size": ("INT", {"default": 61, "min": 1, "step": 2}),
+                "sigma": ("INT", {"default": 5, "min": 1, "step": 2}),
```

Comment:
I think only `kernel_size` needs to be odd?

I took a look at the source code for the [torchvision docs](https://pytorch.org/vision/main/_modules/torchvision/transforms/transforms.html#GaussianBlur) and it looks like these are the constraints placed on the param values:

```
def __init__(self, kernel_size, sigma=(0.1, 2.0)):
        super().__init__()
        _log_api_usage_once(self)
        self.kernel_size = _setup_size(kernel_size, "Kernel size should be a tuple/list of two integers")
        for ks in self.kernel_size:
            if ks <= 0 or ks % 2 == 0:
                raise ValueError("Kernel size value should be an odd and positive number.")

        if isinstance(sigma, numbers.Number):
            if sigma <= 0:
                raise ValueError("If sigma is a single number, it must be positive.")
            sigma = (sigma, sigma)
        elif isinstance(sigma, Sequence) and len(sigma) == 2:
            if not 0.0 < sigma[0] <= sigma[1]:
                raise ValueError("sigma values should be positive and of the form (min, max).")
        else:
            raise ValueError("sigma should be a single number or a list/tuple with length 2.")

        self.sigma = sigma
```

So, the `kernel_size` constraints could remain the same and we could just remove the `step` param for `sigma` - WDYT?

Reply:
I thought I saw the same error message for both, but I just double checked - good catch! We need non-zero for both and odd for only kernel_size.